tap_sftp/client.py

```diff
@@ -170,4 +197,5 @@ def connection(config):
                           config['username'],
                           password=config.get('password'),
                           private_key_file=config.get('private_key_file'),
-                          port=config.get('port'))
+                          port=config.get('port'),
+                          timeout=config.get('request_timeout'))
```

Comment:
@harshpatel4crest Why have you used `request_timeout` from config directly? Can't we use`self.request_timeout`?

Reply:
Here we are initializing the connection. Hence we are passing the timeout value from the config.